plugins/jobs/src/js/data/JobModel.ts

```diff
@@ -23,8 +25,8 @@ export interface JobsQueryArgs {
 
 export interface JobQueryArgs {
   id: string;
-  filter?: string | null;
-  sortBy?: SortOption | null;
+  filter?: string;
```

Comment:
Why did you remove the null?

Reply:
I think that was also an attempt to fix the same problem we discussed above.

But on another pass, I don't think the null should be necessary. I think by saying the parameter value is optional, you pass it or not (I know that is just covering the undefined). I see no point in assigning null to the value if for the filter, or for the sorting, the value of null has no specific meaning.

Reply:
yeah, I think that's fine with me 🤷‍♂️

Reply:
I changed that because it helps me solve the `blabla || null`